kani_metadata/src/unstable.rs

```diff
@@ -83,6 +83,8 @@ pub enum UnstableFeature {
     SourceCoverage,
     /// Enable function contracts [RFC 9](https://model-checking.github.io/kani/rfc/rfcs/0009-function-contracts.html)
     FunctionContracts,
+    /// Enable loop contracts [RFC 12]
```

Comment:
Please add the link.

BTW, will this enable the entire flow where a loop is checked and later replaced?

Reply:
Yes, this will enable the entire flow. The two test cases run with loop contracts enable, where the running time is only ~1 seconds comparing to timeout when running without loop contracts